pyproject.toml

```diff
@@ -1,20 +1,26 @@
 [tool.poetry]
 name = "peopledatalabs"
-version = "1.0.2"
```

Comment:
@vvillait88 also notice I'm changing version. That will be necessary, correct?

Reply:
yeah

Reply:
i think this also needs updating: https://github.com/peopledatalabs/peopledatalabs-python/blob/main/src/peopledatalabs/__init__.py#L9